arangod/RestServer/BootstrapFeature.cpp

```diff
@@ -317,7 +327,7 @@ void BootstrapFeature::start() {
     if (ServerState::isSingleServer(role) && AgencyCommManager::isEnabled()) {
       ::runActiveFailoverStart(myId);
     } else {
-      ss->setFoxxmaster(myId);  // could be empty, but set anyway
+      ServerState::instance()->setFoxxmaster(myId);  // could be empty, but set anyway
```

Comment:
can `ServerState::instance()` return nullptr?

Reply:
No, it can never be a nullptr:
```
static ServerState Instance;
...
ServerState* ServerState::instance() { return &Instance; }
```